src/main/java/it/smartcommunitylab/aac/webauthn/persistence/WebAuthnCredentialsRepository.java

```diff
@@ -0,0 +1,16 @@
+package it.smartcommunitylab.aac.webauthn.persistence;
+
+import com.yubico.webauthn.data.ByteArray;
+
+import org.springframework.stereotype.Repository;
+
+import it.smartcommunitylab.aac.repository.CustomJpaRepository;
+import it.smartcommunitylab.aac.repository.DetachableJpaRepository;
+
+@Repository
+public interface WebAuthnCredentialsRepository
+        extends CustomJpaRepository<WebAuthnCredential, ByteArray>, DetachableJpaRepository<WebAuthnCredential> {
```

Comment:
id in webauthncredential entity is defined as string, not as bytearray.

Reply:
Fixed in 65b7928ea1905376322ddacd9ca397788d97b2c0